<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1665</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="blueSliderMin">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>60</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="blueSliderMax">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>60</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="greenSliderMin">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>170</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="greenSliderMax">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>170</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="redSliderMin">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>280</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="redSliderMax">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>280</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>10</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>240</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>270</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>270</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>160</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>160</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>50</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>50</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="redBoxMin">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>310</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="redBoxMax">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>310</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="greenBoxMax">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>200</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="greenBoxMin">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>200</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="blueBoxMax">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>90</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="blueBoxMin">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>90</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>6</y>
      <width>1291</width>
      <height>1001</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../pictureSet1_1.png_red.png</pixmap>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>360</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter on color</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1665</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>blueSliderMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueBoxMin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>797</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>790</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSliderMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueBoxMax</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>960</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>964</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSliderMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenBoxMin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>800</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>793</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSliderMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenBoxMax</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>957</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>957</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSliderMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redBoxMin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSliderMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redBoxMax</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>975</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>975</x>
     <y>448</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
